templates/pages/about.html

```diff
@@ -8,27 +8,59 @@
 
 
 	<div class="jumbotron">
-		<p>We, CodeCatz, are learning to code, beause we can and because we want to! </p>
-		<p>CodeCatz is a group of programming kittens and several slightly more experienced cats. Kittens want to learn coding, cats are here to help them. We started with Python and web applications, we'll continue with whatnots.</p>
-		<p> Who knows where we will end! </p>
+		<h2 class="about">About CodeCatz</h2>
+		<p>
```

Comment:
to create empty line between the paragraphs. I understand this is  not a good practice but I do not know how to do it differently. As soon as I learn I plan to correct this.